dev-aws/kafka-shared-msk/ledgers/ledger.tf

```diff
@@ -0,0 +1,50 @@
+# Topics
+resource "kafka_topic" "account_balance_change_v3" {
+  name               = "ledgers.account-balance-change-v3"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # store data zstd compressed
+    "compression.type" = "zstd"
+    # Use tiered storage
+    "remote.storage.enable" = "true"
+    # keep data in primary storage for 2 days
+    "local.retention.ms" = "172800000"
+    # keep on each partition unlimited data
+    "retention.bytes" = "-1"
```

Comment:
Do we need this?

Reply:
Not sure, so I removed. We can add it back if we need it